x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -239,7 +239,9 @@ export function App({
                 setState(s => ({ ...s, savedQuery }));
               }}
               onSavedQueryUpdated={savedQuery => {
-                data.query.filterManager.setFilters(savedQuery.attributes.filters || state.filters);
```

Comment:
@wylieconlon do you mind looking at the Lens changes to make sure I didn't screw anything up? It's just this file and the test file above.